DESCRIPTION

```diff
@@ -13,9 +13,13 @@ Imports:
   bootnet,
   corpcor,
   dplyr,
+  tidyr,
+  Matrix, 
+  xfun,
```

Comment:
do we need these? I didn't see them being used in the code?

Reply:
Yes you are right! I think I used them before but forgot to delete them